app/postinstallscript.sh

```diff
@@ -1,7 +1,9 @@
 #!/bin/sh -e
 
-# *** root user required ****
-# TODO Add a check of who the user is and log warning if not root
+if [ "$(id -un)" != "root" ]; then
+    logger -p user.warn "$0: Must be run as 'root' instead of user '$(id -un)'."
```

Comment:
nice with message to syslog but the context will be [<user>] - it needs to be clearer where it comes from or we might end up with a lot of unhappy QA people